### PR TITLE
refine GLM

### PR DESCRIPTION
**Pull request by yuzhou03**
1. add 4 time and 3 perf
2. update case readme
3. GLM precision: TF32 default precision for nvidia ampere arch GPU
 MFU = (2f+6b)\*token_per_second\*p_core / (N * FLOPS) = 6\*0.33\*10^9\*512\*43.5/(8\*156\*10^12) = 0.035

### CODE REVIEW

**Comment by yuzhou03** [comment]
1x1, 1x8, 1x8 big logs included.

[glm_nvidia_training_logs.zip](https://github.com/FlagOpen/FlagPerf/files/12287627/glm_nvidia_training_logs.zip)

**Comment by shh2000** [comment]
MFU算的不对。torch对安培卡默认用tf32，32-bit flops=156T

**Comment by shh2000** [comment]
> MFU算的不对。torch对安培卡默认用tf32，32-bit flops=156T

可以用torch.backends.cuda.<aten/prim::op>.allow_tf32确认model中的tf32比例，通常接近100%

**Review by shh2000 — APPROVED** [review]

File: training/nvidia/glm-pytorch/README.md [hunk 1]
```diff
@@ -18,10 +18,31 @@
...
+| 训练结果         | acc,见“性能指标”                             | 准确率                                      |
+| 额外修改项       | 无                                           |                                             |
+
+* 性能指标
+
+| 配置                | precision | fix_hp          | e2e_time | p_whole | p_train | p_core | acc   | mem       | MFU   |
+| ------------------- | --------- | --------------- | -------- | ------- | ------- | ------ | ----- | --------- | ----- |
+| A100单机8卡（1x8）  | fp32      | /               | 2763     | 36.5    | 42.4    | 42.4   | 0.808 | 33.0/40.0 | 0.035 |
+| A100单机8卡（1x8）  | fp32      | bs=16, lr=1e-05 | 2688     | 37.4    | 43.5    | 43.5   | 0.801 | 39.5/40.0 | 0.035 |
+| A100单机单卡（1x1） | fp32      | bs=16, lr=1e-05 |          | 0.35    | 5.5     | 5.5    |       | 35.0/40.0 |       |
```
Comment (upvenly) [inline training/nvidia/glm-pytorch/README.md:46]: 这个数据为啥缺失

Comment (yuzhou03) [reply training/nvidia/glm-pytorch/README.md]: 已补充

**Review by upvenly — APPROVED** [review]

**Pull request merged**